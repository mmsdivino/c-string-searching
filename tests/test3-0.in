O Saci-Perere e uma lenda do folclore brasileiro e originou-se entre as tribos indigenas do sul do Brasil.

O saci possui apenas uma perna, usa um gorro vermelho e sempre esta com um cachimbo na boca.
Inicialmente, o saci era retratado como um curumim endiabrado, com duas pernas, cor morena, alem de possuir um rabo tipico.

Com a influencia da mitologia africana, o saci se transformou em um negrinho que perdeu a perna lutando capoeira, alem disso, herdou o pito, uma especie de cachimbo, e ganhou da mitologia europeia um gorrinho vermelho.
A principal caracteristica do saci e a travessura, ele e muito brincalhao, diverte-se com os animais e com as pessoas. Por ser  muito moleque ele acaba causando transtornos, como: fazer o feijao queimar, esconder objetos, jogar os dedais das costureiras em buracos e etc.

Segundo a lenda, o Saci esta nos redemoinhos de vento e pode ser capturado jogando uma peneira sobre os redemoinhos.
Apos a captura, deve-se retirar o capuz da criatura para garantir sua obediencia e prende-lo em uma garrafa.
Diz tambem a lenda que os Sacis nascem em brotos de bambus, onde vivem sete anos e, apos esse tempo, vivem mais setenta e sete para atentar a vida dos humanos e animais, depois morrem e viram um cogumelo venenoso ou uma orelha de pau.

In information theory and computer science, the Levenshtein distance is a string metric for measuring the difference between two sequences. Informally, the Levenshtein distance between two words is the minimum number of single-character edits - i.e. insertions, deletions or substitutions - required to change one word into the other. It is named after Vladimir Levenshtein, who considered this distance in.

Levenshtein distance may also be referred to as edit distance, although that may also denote a larger family of distance metrics. It is closely related to pairwise string alignments.
