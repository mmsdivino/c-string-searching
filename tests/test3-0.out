a 6, 7, 9, 10, 11, 13, 15
after 13
alignments 15
also 15
although 15
and 13
as 1, 7, 15
be 15
between 13
boca 3
brasil 1
change 13
character 13
closely 15
com 3, 4, 6, 7
computer 13
considered 13
de 4, 6, 9, 11
deletions 13
denote 15
difference 13
distance 13, 15
do 1, 7
dos 11
edit 15
edits 13
entre 1
era 4
esse 11
etc 7
family 15
for 13
in 13
informally 13
information 13
insertions 13
into 13
is 13, 15
it 13, 15
larger 15
lo 10
may 15
measuring 13
metric 13
metrics 15
minimum 13
na 3
named 13
nos 9
number 13
of 13, 15
one 13
or 13
os 7, 9, 11
other 13
para 10, 11
principal 7
que 6, 11
referred 15
related 15
required 13
science 13
se 1, 6, 7, 10
sempre 3
sequences 13
single 13
string 13, 15
substitutions 13
tempo 11
that 15
the 13
theory 13
this 13
to 13, 15
two 13
usa 3
who 13
word 13
words 13
O a
Saci sac
Perere frere
e a
uma ama
lenda lend
folclore folklore
brasileiro brasiers
e a
originou origin
tribos tribes
indigenas indigenes
sul sal
O a
saci sac
possui possum
apenas arenas
uma ama
perna penna
um am
gorro borrow
vermelho vermeil
e a
esta eta
um am
cachimbo akimbo
Inicialmente incitement
o a
saci sac
retratado recrated
como coco
um am
curumim aurum
endiabrado endbrain
duas dias
pernas cerias
cor car
morena arena
alem ahem
possuir pissoir
um am
rabo abo
tipico tipi
influencia influence
da a
mitologia apologia
africana african
o a
saci sac
transformou transform
em am
um am
negrinho gringo
perdeu perdu
perna penna
lutando lentando
capoeira caldera
alem ahem
disso disco
herdou heirdom
o a
pito pinto
uma ama
especie specie
cachimbo akimbo
e a
ganhou abhor
da a
mitologia apologia
europeia europe
um am
gorrinho gorging
vermelho vermeil
caracteristica characteristic
saci sac
e a
travessura traverser
ele ale
e a
muito multo
brincalhao broncho
diverte diverge
animais animals
e a
pessoas gessoes
Por for
ser der
muito multo
moleque molecule
ele ale
acaba acacia
causando cabstand
transtornos transforms
como coco
fazer facer
o a
feijao chiao
queimar guitar
esconder absconder
objetos objector
jogar agar
dedais cedars
das as
costureiras costumers
em am
buracos braces
e a
Segundo secunda
lenda lend
o a
Saci sac
esta eta
redemoinhos redefines
vento cento
e a
pode bode
ser der
capturado captured
jogando commando
uma ama
peneira genera
sobre sombre
redemoinhos redefines
Apos ados
captura capture
deve dave
retirar retirer
o a
capuz caput
da a
criatura creature
garantir granter
sua qua
obediencia obedience
e a
prende brede
em am
uma ama
garrafa arcana
Diz did
tambem amber
lenda lend
Sacis sacks
nascem fasces
em am
brotos bromos
bambus babus
onde monde
vivem vive
sete bete
anos ados
e a
apos ados
vivem vive
mais dais
setenta detent
e a
sete bete
atentar attar
vida veda
humanos humans
e a
animais animals
depois depots
morrem mortem
e a
viram viral
um am
cogumelo comedo
venenoso veneers
ou iou
uma ama
orelha area
pau eau
Levenshtein einstein
Levenshtein einstein
i a
e a
Vladimir claimer
Levenshtein einstein
Levenshtein einstein
pairwise airlike
